public/blog.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<title>blog</title>
+
+<meta name="Sally Moore - blog" content="Web developer with a Ph.D. in clinical psychology. Likes puppies." charset="UTF-8">
+</head>
+
+<h1>blog</h1>
+
+<header>
+  <a href="/">home</a>&nbsp;&nbsp;&nbsp;
+  <a href="/about">about me</a>&nbsp;&nbsp;&nbsp;
+  <a href="/projects">projects</a>&nbsp;&nbsp;&nbsp;
+  <a href="/pubs">publications in psych</a>
+</header>
+<br>
+
+<body>
+  <img src="/images/adacard.jpg" alt="Go Sounders!" height="250" width="250">
+
+  <p>
+  Recently, I've been writing a series of articles for GeekWire chronicaling my ADA adventure. This has become my de facto blog. Check it out:<br>
+  </p>
+  <br>
+    <article>
+      <h4><a href="http://www.geekwire.com/2015/this-woman-codes-my-path-into-tech/">ch[0] - This woman codes</a></h4>
+      <p>Describes the process of deciding to leave my role as a clinical psychologist to learn to code.</p>
+    </article>
+    <br>
+
+    <article>
+      <h4><a href="http://www.geekwire.com/2015/learning-to-code-self-acceptance-and-star-trek/">ch[1] - Self-acceptance and <em>Star Trek</em></a></h4>
+      <p>About the application process. I'm glad that's over with!</p>
+    </article>
+    <br>
+
+    <article>
+      <h4><a href="http://www.geekwire.com/2015/the-interview-the-pressure-builds-to-get-into-ada-developers-academy/">ch[2] - The interview</a></h4>
+      <p>This one is about the interview process and learning that I was accepted to cohort[2]. Yay!!!!</p>
+    </article>
+    <br>
+</body>
+
+<footer>
```

Comment:
I foolishly forgot to cover how the `<body>` tag works in our initial conversation about html. To truly be considered part of the page, the `<footer>` block here needs to be inside the `<body>` tag. The general rule is that everything in `<head>` is supporting the page without being rendered (meta information, browser info, etc), and anything that should be visible to the user should be nested in the `<body>` tag.